eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/Models/ExtendedLocation.Serialization.cs

```diff
@@ -13,7 +13,7 @@
 using Azure.Core;
 using MgmtTypeSpec;
 
-namespace MgmtTypeSpec.Models
+namespace Azure.ResourceManager.CommonTypes.Models
 {
     /// <summary></summary>
     public partial class ExtendedLocation : IJsonModel<ExtendedLocation>
```

Comment:
Most of the models under `Azure.ResourceManager.CommonTypes` will be replaced by models from `Azure.ResourceManager` in https://github.com/microsoft/typespec/issues/5480. And the ones not referenced in the public models will be omitted.